src/components/treeInfo/index.tsx

```diff
@@ -63,6 +76,25 @@ const TreeInfo: React.FC<TreeProps> = ({
     }
   };
 
+  const shareButton = (
+    <ShareButton
+      size={mobile ? 'middle' : 'large'}
+      // if user owns the tree, show first line
+      // otherwise if someone else has adopted, show second line
+      // otherwise nobody has adopted the tree - show final line
+      defaultText={
+        userOwnsTree
+          ? 'Check out this tree I adopted!'
+          : adopted
+          ? 'Check out this tree near you!'
```

Comment:
nit: Maybe this should be resolved in the ticket, but I think "Adopt this tree near you!" might be better.

Reply:
not sure which case you're talking about - if you mean the second line ("Check out this tree near you!"), that's for trees that someone else has already adopted

Reply:
Ah that makes sense.